### PR TITLE
🙊Feast helm charts🙊

### PR DESCRIPTION
**Pull request by ckavili**
I have questions @RHRolun :D

### CODE REVIEW

**Comment by RHRolun** [comment]
@ckavili The model [here](https://github.com/rhoai-mlops/mlops-helmcharts/blob/main/charts/pipelines/templates/tasks/build-modelcar.yaml#L28) is placed in models/1/... instead of models/jukebox/1/... - is this intentional? :) 
In that case I'll just point the scaler_file_path and encoder_file_path to reflect that.

**Comment by ckavili** [comment]
ah no, it's my mistake but apparently, ovms can understand that too 😅 but to make it more readable I updated it [here](https://github.com/rhoai-mlops/mlops-helmcharts/pull/18/files#diff-1ffc82c88ca1835955927a3b85ecded86eb1ce37bcc78d3c60c451a9e66757e0R27) thanks for catching that 🙈

**Comment by ckavili** [comment]
> @ckavili The model [here](https://github.com/rhoai-mlops/mlops-helmcharts/blob/main/charts/pipelines/templates/tasks/build-modelcar.yaml#L28) is placed in models/1/... instead of models/jukebox/1/... - is this intentional? :) In that case I'll just point the scaler_file_path and encoder_file_path to reflect that.

ah no, it's my mistake but apparently, ovms can understand that too 😅 but to make it more readable I updated it [here](https://github.com/rhoai-mlops/mlops-helmcharts/pull/18/files#diff-1ffc82c88ca1835955927a3b85ecded86eb1ce37bcc78d3c60c451a9e66757e0R27) thanks for catching that 🙈

**Comment by ckavili** [comment]
by the way since you are at it, can you also update [this one](https://github.com/rhoai-mlops/mlops-helmcharts/blob/main/charts/model-deployment/simple/templates/inferenceservice.yaml#L6) as we don't need RawDeployment anymore? thaaaaanks 🙌🤭

File: charts/model-deployment/music-transformer-with-feast/templates/inferenceservice.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          - '--scaler_file_path=models/{{ .Values.name }}/1/artifacts/scaler.pkl'
+          - '--encoder_file_path=models/{{ .Values.name }}/1/artifacts/label_encoder.pkl'
+          - '--feast_server_url={{ .Values.feast_server_url }}'
+          - '--feature_service={{ .Values.feature_service }}'
+          - '--entity_id_name={{ .Values.entity_id_name }}'
+        command:
+          - python
+          - '-m'
+          - music_transformer
+        image: 'quay.io/rlundber/feast_music_transformer:0.6'
```
[the remark this answers is not in the document]
Comment (RHRolun) [reply charts/model-deployment/music-transformer-with-feast/templates/inferenceservice.yaml]: Thanks for creating, they have been moved there :)

File: charts/model-deployment/music-transformer/templates/servingruntime.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
     prometheus.io/port: '8888'
   containers:
     - args:
-        - '--model_name={{.Name}}'
+        - '--model_name={{ .Values.name }}'
```
Comment (ckavili) [inline charts/model-deployment/music-transformer/templates/servingruntime.yaml:21]: hold on sorry, this one can stay as `{{.Name}}` , my mistake 😅

```suggestion
        - '--model_name={{.Name}}'
```

**Pull request merged**